profiles/events.py

```diff
@@ -1,11 +1,5 @@
 import logging
-from typing import (
-    Any,
-    Callable,
-    Dict,
-    List,
-    Tuple,
-)
+from typing import Any, Callable, Dict, List, Tuple
```

Comment:
Fine to revert this, but I consciously made the change as its seen to be idiomatic to wrap 3 or more imports in a single line into a tuple, e.g. 

https://github.com/django/django/blob/master/django/http/multipartparser.py

It is not enforced by the linters (as far as I know) but is common place in major projects/libraries.

Reply:
Ah ok, was just aiming for a smaller diff (and it hasn't been done elsewhere).

Reply:
OK granted, there are a few files this could be applied to. Just leave it for now then.